error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/TimeRules.java

```diff
@@ -64,8 +66,67 @@ ZoneOffset after() {
     }
   }
 
+  static final class InstantToLocalDate {
```

Comment:
We generally name the templates after the "shape" of the code in the `@AfterTemplate`s (with the idea that eventually we'll automate this, because naming is hard). Will propose something :)